### ISSUE TITLE
adding the --proxy option so as to debug requests

### ISSUE THREAD

**Issue by mailler0xa** [grampae/VMSA-2023-0017#1]
Hello, I have been trying to debug the requests to figure out how to proxy the requests through burpsuite yet I can't get it quite right. Is it possible to give me pointers on how to do it?

Regards.

**Comment by grampae** [comment]
Yeah of course, so you can add the following within the script assuming your Burp is running on 8080
`proxies = { 'http': 'http://127.0.0.1:8080', 'https': 'http://127.0.0.1:8080' } `

Then anywhere requests is used for example
`r = s.post(url1, headers=sheaders, data=sdata, verify=False, allow_redirects=False, timeout=10)`
you can add "proxies=proxies"  so it will look like.
`r = s.post(url1, headers=sheaders, data=sdata, verify=False, allow_redirects=False, timeout=10, proxies=proxies)`
